ticketutil/bugzilla.py

```diff
@@ -0,0 +1,301 @@
+import logging
+
+import requests
+
+from . import ticket
+
+__author__ = 'dranck, rnester, kshirsal'
+
+# Disable warnings for requests because we aren't doing certificate verification
+requests.packages.urllib3.disable_warnings()
+
+DEBUG = False
```

Comment:
Added:
To enable debug logging for ticketutil, set an environment 
variable named TICKETUTIL_DEBUG to 'True'. If this environment variable
is set to anything else or does not exist, debug logging will be 
disabled.